Automattic/jetpack
# Blogroll : Block refactor, styling changes, and additional enhancements

Fixes:
https://github.com/Automattic/wp-calypso/issues/81351

## Proposed changes:


Fix default icon image
Fix icon styling for width and height
Remove Blogroll settings and setting context values

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to  the editor
* Add the blogroll block
* Ensure that the response from `recommendations` is used to create the initial content.
* Ensure that the suggestions are returning correctly.
